server/rooms.js

```diff
@@ -1693,12 +1693,18 @@ let Rooms = Object.assign(getRoom, {
 				user.inviteOnlyNextBattle = false;
 			}
 		}
-		if (options.tour && !room.tour.modjoin) inviteOnly = [];
 		if (inviteOnly.length) {
-			room.modjoin = '%';
-			room.isPrivate = 'hidden';
-			room.privacySetter = new Set(inviteOnly);
-			room.add(`|raw|<div class="broadcast-red"><strong>This battle is invite-only!</strong><br />Users must be invited with <code>/invite</code> (or be staff) to join</div>`);
+			const prefix = /** @type {RoomBattle} */(room.game).forcedPublic();
+			if (prefix && room.rated) {
```

Comment:
```suggestion
			if (prefix && room.game.rated) {
```

Reply:
I switched to this, but I dont think it should matter? `GameRoom` also has a `rated` field based on `options.rated`, no? It should be pretty much the exact same thing?

Reply:
It /shouldn't/ matter, but it's a bit more thinking to arrive at the same conclusion.

`room` is `createGameRoom` which calls `GameRoom` which has a `rated` property.

vs.

`room.game` is `RoomBattle` which has a rated property.

`room` also isn't usually associated with rated-ness.

Super nitpick, but yeah.